x-pack/plugins/enterprise_search/public/applications/app_search/app_logic.ts

```diff
@@ -16,6 +16,7 @@ export interface IAppActions {
 }
 
 export const AppLogic = kea<MakeLogicType<IAppValues, IAppActions>>({
+  path: ['enterpriseSearch', 'appSearch', 'app'],
```

Comment:
@JasonStoltz changed from snake_case to camelCase, hope that's cool 🤔 

Also... do we have any strong thoughts about using our `common/constants.ts` strings here? In theory this could be:

```js
path: [ENTERPRISE_SEARCH_PLUGIN.ID, APP_SEARCH_PLUGIN.ID, 'app']
```

instead... but I don't know if that's necessarily better/easier to read lol? (I'm pretty lazy about constant strings when it comes to dev-only tbh though, it's one of my many failings)

Reply:
tbh I like camel case because it matches the directory/file structure.  If I saw `enterprise_search -> app_search -> app_logic` I know exactly what I need to type into the file jumper in vscode to get there.  Otherwise if we use camelcase these should be names of the components and Logics [`EnterpriseSearch`, `AppSearchConfigured`, `AppLogic`].  If this is meant to ease debugging it should give me something I can easily search our codebase for to get to.

Reply:
Doesn't matter to me, I just want to get this merged

Reply:
> tbh I like camel case

assuming you mean snake case, just to confirm?

re: names lining up with directory / file names:

I think if we want to guarantee that, we should just install [babel-plugin-kea](https://kea.js.org/docs/guide/debugging/#automatic-paths-with-babel) to automatically handle that for us. Otherwise, there's absolutely no guarantee that someone will name all logic files correctly in the future (for example, this line in question is `app`, not even `appLogic`).

I can definitely switch this plugin over to just being yarn install if we're cool with it. Yay/nay?

Reply:
> assuming you mean snake case, just to confirm? 

yeah sorry, I'm a idiot lmao

10000% for using this babel plugin

Reply:
Awesome possum. Imma do it here shortly

Reply:
After a bit more digging: sadly it's not that straightforward 😞 Since Kibana maintains it's own [babel package/settings](https://github.com/elastic/kibana/tree/master/packages/kbn-babel-preset), we'll have to open an issue and get approval from the ops team after they evaluate the plugin.

Additionally, after looking at the plugin source code, it [camelCases file names](https://github.com/keajs/babel-plugin-kea/blob/master/src/index.js#L63) so it actually wouldn't achieve what Byron wants in any case (doh).

@byronhulcher it sounds like you have the strongest feelings here re: path names, so I'll go ahead and rename to snake_case as well as matching file names exactly. I'm also going to update our documentation/README to note our use of Kea and path names, and why we're specifically using snake_case for those lines.